content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -19,7 +19,7 @@ This document catalogs the communication paths between the control plane (really
 {{% capture body %}}
 
 ## Node to Control Plane
-All communication paths from the nodes to the control plane terminate at the apiserver (none of the other master components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
+All communication paths from the nodes to the control plane terminate at the apiserver (none of the other master components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (6443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
```

Comment:
"master components" should be "control-plane components"

Reply:
Done, thanks.